### PR TITLE
chore: increase the coverage in BasicKV

### PR DESCRIPTION
**Pull request by SYaoJun**

### CODE REVIEW

File: tests/BasicKVTest.cpp [hunk 1]
```diff
@@ -133,4 +133,30 @@ TEST_F(BasicKVTest, BasicKVInsertAndLookup) {
...
+    btree = res.value();
+    cr::Worker::My().StartTx();
+    EXPECT_EQ(btree->Insert(Slice((const uint8_t*)key.data(), key.size()),
+                            Slice((const uint8_t*)val.data(), val.size())),
+              OpCode::kOK);
+    EXPECT_EQ(btree->Insert(Slice((const uint8_t*)key.data(), key.size()),
+                            Slice((const uint8_t*)val.data(), val.size())),
+              OpCode::kDuplicated);
+    cr::Worker::My().CommitTx();
+  });
```
Comment (zz-jason) [inline tests/BasicKVTest.cpp:158]: how about adding one more test to check the key-value pair inserted in the btree?

File: tests/BasicKVTest.cpp [hunk 2]
```diff
@@ -133,4 +133,30 @@ TEST_F(BasicKVTest, BasicKVInsertAndLookup) {
   });
 }
 
+TEST_F(BasicKVTest, BasicKVInsertDuplicatedKey) {
+  // create leanstore btree for table records
+  storage::btree::BasicKV* btree;
+  const auto* btreeName = "testTree1";
+
+   std::string key("key_btree_LL_xxxxxxxxxxxx_");
```
Comment (zz-jason) [inline tests/BasicKVTest.cpp:141]: the code format seems to be incorrect, you can fix it by `cmake --build build/debug --target=format`

Comment (SYaoJun) [reply tests/BasicKVTest.cpp:141]: ok

File: tests/BasicKVTest.cpp (hunk 1 above)
Comment (SYaoJun) [reply tests/BasicKVTest.cpp:158]: > how about adding one more test to check the key-value pair inserted in the btree?

Are you suggesting that I should use the lookup() function to verify if the key-value pair has been inserted, or should I also check this in another worker? Or do both conditions need to be verified?

Comment (zz-jason) [reply tests/BasicKVTest.cpp:158]: Check it in the same worker is fine. You can also check it in another worker.

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**